rundeckapp/grails-app/migrations/core/ConstraintsIndexesKeys.groovy

```diff
@@ -543,6 +512,7 @@ databaseChangeLog = {
             not{
                 foreignKeyConstraintExists (foreignKeyTableName: "node_filter", foreignKeyName: "FKhuj3n3gy2kqoie8d8oucay4b3")
             }
+            tableExists (tableName:"node_filter")
         }
         addForeignKeyConstraint(baseColumnNames: "user_id", baseTableName: "node_filter", constraintName: "FKhuj3n3gy2kqoie8d8oucay4b3", deferrable: "false", initiallyDeferred: "false", referencedColumnNames: "id", referencedTableName: "rduser", validate: "true")
```

Comment:
i think this changeset is no longer needed

Reply:
removed